pkg/foundation/ctxutil/transaction.go

```diff
@@ -17,7 +17,7 @@ package ctxutil
 import (
 	"context"
 
-	"github.com/conduitio/conduit/pkg/foundation/database"
+	"github.com/conduitio/conduit-commons/database"
 )
 
 // transactionCtxKey is used as the key when saving the transaction in a context.
```

Comment:
We should also remove this file and use the corresponding functions in the `database` package instead.